### ISSUE TITLE
"tags" and "categories" features

### ISSUE THREAD

**Issue by linlennypinawa** [rstudio/bookdown#1375]
I like "tags" and "Category" features in YAML header of *.md for building blogs. Do you think it can be implemented in *.rmd file for bookdown?

**Comment by cderv** [comment]
Can you specify a bit more of context about the use case ? 

**bookdown** is a R Markdown format to write Books not blog. See https://bookdown.org/yihui/bookdown/

Are you referring to **blogdown** instead ? https://bookdown.org/yihui/blogdown/ 

YAML header value in **blogdown** Rmd post will depend on the Hugo theme used mainly. Tags and categories should be compatible with some. 

**distill** is another R Markdown format which helps building blogs: https://rstudio.github.io/distill/blog.html

As you see, your question if for now a bit vague for us to help with. 

thank you

**Comment by linlennypinawa** [comment]
I built a lab logbook with the`hugo-book` theme, and put [it](https://troubleshooting-logbook.netlify.app/) online for the promotion purpose.  Eventually it would be offline in practice.  The reason is that not all business wants to put their info online.

I think the offline option would be a bookdown project.  I like the `tags`  and `categories` features in the `hugo-book` theme.  They help me to organize *.md files in different ways even though they are under different directories.  I wish my bookdown project would have such features.

**Comment by cderv** [comment]
I don't think this feature is already in one of the `gitbook()` or `bs4_book()` format. Unfortunately, we won't probably add such feature in the future are those format are hard too tweaks for such use case between HTML books, and Websites. 

This is also because we've been contributing to a new project called Quarto which extend the R Markdown ecosystem, and started from scratch recently trying to build on all the best from what have been done in the past. 

This project is called Quarto (https://quarto.org/). You should have a look because it offers nice website / blog features, and listing pages are one of them (https://quarto.org/docs/websites/website-listings.html). This allows to organize content differently with categories and tags as you mentioned. You should give it a try if you feel too limited by what **bookdown** offers. 

Thanks !
